_posts/blog/2016-08-21-gopherjs-1.7-1-release.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "GopherJS 1.7-1 is released"
+date:   2016-08-21
+author: Dmitri Shuralyov
+---
+
+Hello gophers,
+
+Today, we are happy to make two announcements for GopherJS. First, GopherJS now [officially supports Go 1.7](https://github.com/gopherjs/gopherjs/issues/475), which is the current stable release of Go that came out recently. Second, GopherJS has its first named version attached to it, and we're calling it GopherJS 1.7-1. This makes it possible to have a release history tracking major changes and additions between versions.
```

Comment:
Ok, agree. I wanted the link to be an optional thing for people to click on to find more background information, but:

- Instead of adding to the message, it dilutes it and distracts the reader.
- Motivated readers can still find the relevant issue and background info if they wish.

I don't think I'll use bold, as that may seem like a "cheap trick". Instead, by removing the link, and having this message delivered in the 2nd sentence, I hope it'll be very visible and seen by readers without having to resort to bolding.

I'll give it and try and we can see how it looks.